doc/api/fs.md

```diff
@@ -2559,8 +2566,9 @@ Synchronous versions of [`fs.write()`][]. Returns the number of bytes written.
 
 ## FS Constants
 
-The following constants are exported by `fs.constants`. **Note:** Not every
-constant will be available on every operating system.
+The following constants are exported by `fs.constants` or `fs` (in case of Node
+`< v6.3.0`). **Note:** Not every constant will be available on every
+operating system.
```

Comment:
Can you also change `in case of` -> `for` (in addition to @jasnell's suggested changes)?